src/common/math_utils.rs

```diff
@@ -27,3 +28,24 @@ impl From<Vector3> for common::v1::Vector3 {
         }
     }
 }
+// If revolutions is 0, the returned wait duration will be 0 representing that
+// the motor should run indefinitely.
+pub(crate) fn go_for_math(max_rpm: f64, rpm: f64, revolutions: f64) -> Result<(f64, Duration), ()> {
```

Comment:
I don't see any case where this returns an Error, can we just return the tuple instead of a Result?

Reply:
now errors of either rpm or revolutions are NaN